provisioner/clusterpy.go

```diff
@@ -184,7 +180,7 @@ func (p *clusterpyProvisioner) propagateConfigItemsToNodePools(cluster *api.Clus
 // Provision provisions/updates a cluster on AWS. Provision is an idempotent
 // operation for the same input.
 func (p *clusterpyProvisioner) Provision(ctx context.Context, logger *log.Entry, cluster *api.Cluster, channelConfig channel.Config) error {
-	awsAdapter, updater, nodePoolManager, err := p.prepareProvision(logger, cluster, channelConfig)
+	awsAdapter, updater, _, err := p.prepareProvision(logger, cluster, channelConfig)
```

Comment:
If the `nodePoolManager` is not used from `prepareProvision`, then it can be removed from the function.